src/HotReload.hs

```diff
@@ -24,7 +24,7 @@ wrap hotReloadingPort (a, content) =
     [ "// Expose the Webpack HMR API"
     , "var myDisposeCallback = function() {};"
     , "// simulate the HMR api exposed by webpack"
-    , "var moduleHot = {"
+    , "var module = {"
```

Comment:
I'm worried that something else is called `module` and we overwrite that.

Reply:
@stoeffel Great idea! I tried using `moduleHot` as before and now it seems to work! Well, it reloads the page. I still get the `renameFile` error... but it's a big progress!